src/js/popup.js

```diff
@@ -1011,12 +1011,15 @@ Logic.registerPanel(P_CONTAINERS_EDIT, {
 
   // This method is called when the object is registered.
   initialize() {
+    // add logic to move between different panel
```

Comment:
@QiiLin 
this is not what we implemented, isn't it?
sure to add comments about old code?